generated/_task_modules/channels/ai_channel.py

```diff
@@ -0,0 +1,202 @@
+# Do not edit this file; it was automatically generated.
+
+import ctypes
+import numpy
+
+from nidaqmx._lib import (
+    lib_importer, wrapped_ndpointer, ctypes_byte_str, c_bool32)
+from nidaqmx.scale import Scale
+from nidaqmx.errors import (
+    check_for_error, is_string_buffer_too_small, is_array_buffer_too_small)
+from nidaqmx._task_modules.channels.channel import Channel
+from nidaqmx.utils import unflatten_channel_string
+
+
+class AIChannel(Channel):
+    """
+    Represents one or more analog input virtual channels and their properties.
+    """
+    __slots__ = []
+
+    def __repr__(self):
+        return 'AIChannel(name={0})'.format(self._name)
+
+
+    @property
+    def ai_atten(self):
+        """
+        Specifies the amount of attenuation to use.
+        """
+        val = ctypes.c_double()
+
+        cfunc = lib_importer.windll.DAQmxGetAIAtten
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.POINTER(ctypes.c_double)]
+
+        error_code = cfunc(
+            self._handle, self._name, ctypes.byref(val))
+        check_for_error(error_code)
+
+        return val.value
+
+    @ai_atten.setter
+    def ai_atten(self, val):
+        cfunc = lib_importer.windll.DAQmxSetAIAtten
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.c_double]
+
+        error_code = cfunc(
+            self._handle, self._name, val)
+        check_for_error(error_code)
+
+    @ai_atten.deleter
+    def ai_atten(self):
+        cfunc = lib_importer.windll.DAQmxResetAIAtten
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str]
+
+        error_code = cfunc(
+            self._handle, self._name)
+        check_for_error(error_code)
+
+
+    @property
+    def ai_raw_samp_justification(self):
+        """
+        Indicates the justification of a raw sample from the device.
+        """
+        val = ctypes.c_int()
+
+        cfunc = lib_importer.windll.DAQmxGetAIRawSampJustification
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.POINTER(ctypes.c_int)]
+
+        error_code = cfunc(
+            self._handle, self._name, ctypes.byref(val))
+        check_for_error(error_code)
+
+        return int(val.value)
```

Comment:
This should return `DataJustification(val.value)` not `int(val.value)`.

![image](https://user-images.githubusercontent.com/17914399/219505410-e19f7cd1-9973-4971-b2fd-f149edb81285.png)

Reply:
Updated the getter template to return the specified `enum` instead of `python_data_type`.